commentHandler.go

```diff
@@ -130,24 +134,84 @@ func handleComment(req types.IssueCommentOuter) {
 	case "close", "reopen":
 		fmt.Printf("%s wants to %s issue #%d\n", req.Comment.User.Login, command.Type, req.Issue.Number)
 
+		newState, validAction := assessState(command.Type, req.Issue.State)
```

Comment:
Maybe call it `isValidAction` / `validTransition` ?

Reply:
Yes

Reply:
It still appears to be `assessState` in the commit comparison for me?

Reply:
I think your original comment was referring to the variable.  If ‘it’ in the original comment was referencing the name of the function, rather than the variable, then the comment should be placed by the function definition, rather than by a particular use of the function.

Reply:
I want to change the name of the function too. I can see what you were going for with `assessState` but would like to pick something else.